Fix example code in Python AWS lambda docs (#18237)
newrelic/docs-website: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/enable-serverless-monitoring-aws-lambda-layerless.mdx

  >
    To instrument your Python Lambda:

>     1. Download our Python agent package as well as the Python lambda wrapper package and place them in the same directory as your function. To do this, use pip:

Reviewer: ```suggestion
    1. Download both the Python agent and Python lambda wrapper packages and place them in the same directory as your function code. To do this, use pip:
```

       ```bash
       pip install -t . newrelic newrelic-lambda
       ```

       <Callout variant="important">
         If you use Homebrew, you may get this error: `DistutilsOptionError: must supply either home or prefix/exec-prefix -- not both`. For details, see the [Homebrew GitHub post](https://github.com/Homebrew/brew/blob/master/docs/Homebrew-and-Python.md#note-on-pip-install---user).
       </Callout>

>     2. In your Lambda code, import the Python agent module and the Python lambda wrapper module. Next, decorate the handler function using the New Relic decorator. <DNT>**The New Relic package must be imported first in your code.**</DNT> Here's an example:

Reviewer: ```suggestion
    2. In your Lambda code, import both the Python agent module and the Python lambda wrapper module. 
    3. Decorate the handler function using the New Relic decorator. The New Relic package must be imported first in your code. Here's an example:
```
